doc/source/user/cli.rst

```diff
@@ -0,0 +1,45 @@
+.. _ug_cli:
+
+***************
+Running FuseSoC
+***************
+
+FuseSoC is a command-line tool; this section explains how to use it.
+The following content is aiming at users who already have a hardware design which uses FuseSoC.
+
+.. todo::
+
+   Think about giving users a demo/tutorial design to download and play with, which is also used in this section.
```

Comment:
We could link to LED to Believe which already support a lot of boards if the users want to try it out, or add their own board otherwise. From LED to Believe there are also links to SERV as a next step

Reply:
This comment was left over from an earlier draft, I removed it for now while we figure out in more detail what a "best practices design" is.